### PR TITLE
feat: specify custom version of Re.Pack to install with `@callstack/repack-init`

### PR DESCRIPTION
**Pull request by jbroma**
### Summary

add `--custom-version` flag to `@callstack/repack-init`. This allows for installing other version than the latest one available on the NPM.

Note: error handling is left to the package manager of choice.

### Test plan

- [x] - tested locally

### CODE REVIEW

File: packages/init/src/bin.ts [hunk 1]
```diff
@@ -11,10 +11,15 @@ const info = require('../package.json');
 
 const argv = yargs(hideBin(process.argv))
   .usage(`Usage: ${info.name} [options]`)
+  .option('custom-version', {
```
Comment (thymikee) [inline packages/init/src/bin.ts:14]: can we change it to just `--version` or will it intercept the version of the program?

Comment (jbroma) [reply packages/init/src/bin.ts:14]: yeah, `--version` was already taken

Comment (thymikee) [reply packages/init/src/bin.ts:14]: so how about `--repack-version` or `--exact-version`? I'm thinking about long-term using custom and it feels ambiguous

**Review by thymikee — APPROVED** [review]
Some docs around available options would be nice

**Comment by jbroma** [comment]
> Some docs around available options would be nice

oh, good point, I forgot to update the README.md inside of `init`. But separate docs for the init should be added too at some point, good place for a contrib 👍

**Pull request merged**